examples/index.ipynb

```diff
@@ -281,11 +281,6 @@
     "Recent release notes are available on [GitHub](https://github.com/ioam/param/releases).\n",
     "\n",
     "For older releases, see our [historical release notes](historical_release_notes.html).\n",
```

Comment:
```suggestion
    "For older releases, see our [historical release notes](historical_release_notes.html).\n"
```

(I think JSON won't allow the trailing comma. In general it's important to load a .ipynb file back into Jupyter any time it is edited directly like this; it's too easy to break the syntax otherwise!)

Reply:
Yes, I am not sure that trailing comma is valid. Safest to remove it.